README.md

```diff
@@ -84,7 +84,7 @@ jobs:
           pandoc: "README.md" # Input pandoc file
           version: "NVIM v0.8.0" # Vim version number
           toc: true # Table of contents
-          description: "" # Project Description
+          description: "" # Project Description that replaces the first line of the vimdoc
```

Comment:
Maybe it should say something like:

> Project Description (if empty, generates a description that shows nvim version and current date)

Thoughts?

Reply:
It's long and AFAIK some people don't like `nvim` instead of `neovim`.

My suggestions:
- Project description used in title (if empty, it's neovim version and current date)
- Project description used in title (by default, it's neovim version and current date)
- Project description used in title (default uses the current neovim version and date)

Your final word?

Reply:
How about this?

> Project description used in title (if empty, uses neovim version and current date)

Reply:
I have updated